`--force` flag is required to run `npm install` in this repository

First reported [here](https://github.com/mozilla/pdf.js/issues/15382#issuecomment-1235314519).

Repro steps:

```text
~/Downloads node --version
v16.15.0
~/Downloads npm --version
8.19.1
~/Downloads git clone https://github.com/mozilla/pdf.js.git
Cloning into 'pdf.js'...
remote: Enumerating objects: 84802, done.
remote: Counting objects: 100% (113/113), done.
remote: Compressing objects: 100% (67/67), done.
remote: Total 84802 (delta 64), reused 74 (delta 46), pack-reused 84689
Receiving objects: 100% (84802/84802), 132.86 MiB | 21.02 MiB/s, done.
Resolving deltas: 100% (60426/60426), done.
~/Downloads cd pdf.js 
~/Downloads/pdf.js (master| ✔) npm i
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: eslint-plugin-mozilla@2.12.4
npm ERR! Found: eslint-plugin-html@7.1.0
npm ERR! node_modules/eslint-plugin-html
npm ERR!   dev eslint-plugin-html@"^7.1.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer eslint-plugin-html@"^6.0.0" from eslint-plugin-mozilla@2.12.4
npm ERR! node_modules/eslint-plugin-mozilla
npm ERR!   dev eslint-plugin-mozilla@"^2.12.4" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: eslint-plugin-html@6.2.0
npm ERR! node_modules/eslint-plugin-html
npm ERR!   peer eslint-plugin-html@"^6.0.0" from eslint-plugin-mozilla@2.12.4
npm ERR!   node_modules/eslint-plugin-mozilla
npm ERR!     dev eslint-plugin-mozilla@"^2.12.4" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /home/srmagura/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/srmagura/.npm/_logs/2022-09-13T01_27_30_763Z-debug-0.log
```

I encoutered same problem as described by @srmagura. Then I tried what @hrittik2002 said, but the "npm uninstall eslint-plugin-html@6.2.0" command failed with almost the same error as "npm install"

````
(master)npm uninstall eslint-plugin-html@6.2.0
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: eslint-plugin-mozilla@2.12.4
npm ERR! Found: eslint-plugin-html@7.1.0
npm ERR! node_modules/eslint-plugin-html
npm ERR!   dev eslint-plugin-html@"^7.1.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer eslint-plugin-html@"^6.0.0" from eslint-plugin-mozilla@2.12.4
npm ERR! node_modules/eslint-plugin-mozilla
npm ERR!   dev eslint-plugin-mozilla@"^2.12.4" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: eslint-plugin-html@6.2.0
npm ERR! node_modules/eslint-plugin-html
npm ERR!   peer eslint-plugin-html@"^6.0.0" from eslint-plugin-mozilla@2.12.4
npm ERR!   node_modules/eslint-plugin-mozilla
npm ERR!     dev eslint-plugin-mozilla@"^2.12.4" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /home/victor/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/victor/.npm/_logs/2022-09-15T10_58_33_252Z-debug-0.log

````

--- a/gulpfile.js
+++ b/gulpfile.js
@@ -2345,7 +2345,7 @@ gulp.task(
       opts.cwd = installPath;
       distPath = path.relative(installPath, distPath);
     }
-    safeSpawnSync("npm", ["install", "--force", distPath], opts);
+    safeSpawnSync("npm", ["install", distPath], opts);
     done();
   })
 );

--- a/package.json
+++ b/package.json
@@ -21,7 +21,7 @@
     "eslint-plugin-html": "^7.1.0",
     "eslint-plugin-import": "^2.27.5",
     "eslint-plugin-json": "^3.1.0",
-    "eslint-plugin-mozilla": "^3.0.1",
+    "eslint-plugin-mozilla": "^3.1.0",
     "eslint-plugin-no-unsanitized": "^4.0.2",
     "eslint-plugin-prettier": "^4.2.1",
     "eslint-plugin-sort-exports": "^0.8.0",

